post_to_mastodon.py

```
@@ -86,15 +92,49 @@
     version=version,
     changelog=changelog,
     issue_url=issue_url,
-    repository_url=repository_url
+    repository_url=repository_url,
 )
 
+# try to extract the release notes from the change log
+# first, we need to find the CHANGELOG.md file
+changelog_path = None
+for root, dirs, files in os.walk(os.getcwd()):
```

Ruff (0.11.9)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)